### ISSUE TITLE
如何可视化热力图？

### ISSUE THREAD

**Issue by shaohailin** [MhLiao/DB#171]

**Comment by sonica1987** [comment]
demo.py 144 行：
            if visualize and self.structure.visualizer:
                vis_image = self.structure.visualizer.demo_visualize(image_path, output)
                hot = pred[0, 0, :, :].detach().cpu().numpy()
                cv2.imwrite(os.path.join(self.args['result_dir'], image_path.split('/')[-1].split('.')[0] + '_Hot.jpg'),
                            hot)
                cv2.imwrite(os.path.join(self.args['result_dir'], image_path.split('/')[-1].split('.')[0]+'.jpg'), vis_image)
